src/cluster.c

```diff
@@ -1189,6 +1211,17 @@ clusterNode *clusterLookupNode(const char *name) {
     return dictGetVal(de);
 }
 
+/* Cluster node name sanity check */
+int verifyClusterNodeName(const char *name, int length) {
```

Comment:
```suggestion
int verifyClusterNodeId(const char *id, int length) {
```
It's an ID, not a name. We have a separate effort to add node names.

Reply:
@madolson  The reason for using `name` is that I noticed that the struct  `clusterNode` has a member variable `char name[CLUSTER_NAMELEN]`. What do you mean about 'a separate effort to add node names'?

Reply:
https://github.com/redis/redis/pull/9564